iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -24,10 +24,9 @@ enum ConversationUIState {
     }
 }
 
-class ConversationListViewController: ViewController,
-                                      UICollectionViewDelegate,
-                                      UICollectionViewDelegateFlowLayout {
```

Comment:
The conversation list vc is no longer for responsible for figuring out which cell is the center. I moved that responsibility to the layout class.